src/fairchem/core/trainers/base_trainer.py

```diff
@@ -455,7 +456,9 @@ def load_model(self) -> None:
             self.logger.log_summary({"num_params": self.model.num_params})
 
         if distutils.initialized() and not self.config["noddp"]:
```

Comment:
How does this PR behave when `--noddp` is defined? Would be good to test that functionality.

Reply:
Added!